modules/@apostrophecms/oembed-field/ui/apos/components/AposInputOembed.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <AposInputWrapper
+    :modifiers="modifiers" :field="field"
+    :error="effectiveError" :uid="uid"
+  >
+    <template #body>
+      <div class="apos-input-wrapper">
+        <input
+          :class="classes"
+          v-model="next.url" type="url"
+          :placeholder="field.placeholder"
+          :disabled="field.disabled" :required="field.required"
+          :id="uid" :tabindex="tabindex"
+        >
+        <component
+          v-if="icon"
+          :size="iconSize"
+          class="apos-input-icon"
+          :is="icon"
+        />
+      </div>
+    </template>
+    <template #secondary>
+      <div
+        v-if="!error && oembedResult.html" v-html="oembedResult.html"
+        class="apos-input__embed" :class="{ 'is-dynamic': !!dynamicRatio }"
+        :style="{ paddingTop: dynamicRatio && `${(dynamicRatio * 100)}%` }"
+      />
+    </template>
+  </AposInputWrapper>
+</template>
+
+<script>
+import AposInputMixin from 'Modules/@apostrophecms/schema/mixins/AposInputMixin';
+
+export default {
+  name: 'AposInputOembed',
+  mixins: [ AposInputMixin ],
+  emits: [ 'return' ],
+  data () {
+    return {
+      next: (this.value && this.value.data !== undefined)
+        ? this.value.data : (this.field.def || ''),
```

Comment:
Fields aren't responsible for this, we set up the defaults at the owner's level now. Even in the widget editor.

Reply:
That's copied from the input mixin.